plugins/telemetry-otel/src/test/java/org/opensearch/telemetry/OTelTelemetryPluginTests.java

```diff
@@ -34,41 +37,47 @@
 
 public class OTelTelemetryPluginTests extends OpenSearchTestCase {
 
-    private OTelTelemetryPlugin oTelTracerModulePlugin;
+    private OTelTelemetryPlugin oTelTelemetryModulePlugin;
```

Comment:
Minor, this is not a module but plugin 

```suggestion
    private OTelTelemetryPlugin oTelTelemetryPlugin;
```